modules/core/src/main/java/org/apache/ignite/mxbean/ThreadPoolMXBean.java

```diff
@@ -143,7 +143,7 @@
      *
      * @return Class name of current rejection handler.
      */
-    @MXBeanDescription("Class name of current rejection handler.")
+    @MXBeanDescription("Handler class name for rejection tasks.")
```

Comment:
Do we need to change the description? RejectedExecutionHandler is actually a java SE class. If you think the change is needed, then
Handler class name for reject_ed_ tasks.